bgpd/bgp_vty.c

```diff
@@ -3769,6 +3769,10 @@ DEFUN (no_neighbor,
 
 			other = peer->doppelganger;
 			peer_notify_unconfig(peer);
+
+			if (CHECK_FLAG(peer->flags, PEER_FLAG_CAPABILITY_ENHE))
+				bgp_zebra_terminate_radv(peer->bgp, peer);
+
 			peer_delete(peer);
 			if (other && other->status != Deleted) {
 				peer_notify_unconfig(other);
```

Comment:
Should we call bgp_zebra_terminate_radv() for  "other" peer as well ?

Reply:
I'll change to a warning and also add the terminate for the "other" peer.